deco-sites/miscelandia
# Develop

## What is this contribution about?

Here are some tweaks:
* purchase btn in dp;
* item quantity selection input in the pdp;
* scroll style;
* some optimizations for performance gain;

## How to test it?

To test it, just visit the homepage, and observe the side scroll and in the pdp now with the component to select the quantity of items to buy
